exo-run/src/service-runner.ls

```diff
@@ -18,7 +18,8 @@ require! {
 class ServiceRunner extends EventEmitter
 
   ({@name, @config, @logger}) ->
-    @service-config = yaml.safe-load (DockerHelper.get-config @config.image)
+    | @config.image  =>  @service-config = yaml.safe-load DockerHelper.get-config(@config.image)
```

Comment:
A class modifying a variable provided to it is an anti-pattern. Now `@config` is different for everybody else in the app, just because it was provided to a ServiceRunner instance. This means `@config` does not reflect what is in the config file, but also what ServiceRunner things the `image` attribute should be. 

Either leave this as an attribute of this class, as it was before, or extract all the logic for gathering configuration information into a `ServiceConfiguration` class.